webpack/webpack.common.js

```diff
@@ -6,11 +6,11 @@ const { appDirectory, buildDirectory } = require("./filePaths");
 
 module.exports = {
   entry: {
-    bundle: `${appDirectory}/index.js`
+    bundle: `${appDirectory}/index.jsx`
   },
   output: {
     path: buildDirectory,
-    filename: "[name].bundle.js"
+    filename: "build.bundle.js"
```

Comment:
i can not also remember when we agreed to change this filename to build.bundle.js